guides/release/pages.yml

```diff
@@ -1,325 +1,325 @@
       url: "part-1"
       pages:
         - title: "Introduction"
           url: "index"
         - title: "Orientation"
           url: "orientation"
-        - title: "Building Pages"
+        - title: "Construire des Pages"
           url: "building-pages"
-        - title: "Automated Testing"
+        - title: "Tests Automatisés"
           url: "automated-testing"
-        - title: "Component Basics"
+        - title: "Composants Basiques"
           url: "component-basics"
-        - title: "More About Components"
+        - title: "Plus sur les Composants"
           url: "more-about-components"
-        - title: "Interactive Components"
+        - title: "Composants Interactifs"
           url: "interactive-components"
-        - title: "Reusable Components"
+        - title: "Composants Réutilisables"
           url: "reusable-components"
-        - title: "Working With Data"
+        - title: "Travailler avec des Données"
           url: "working-with-data"
-        - title: "Recap"
+        - title: "Récap"
           url: "recap"
-    - title: "Part 2"
+    - title: "Partie 2"
       url: "part-2"
       pages:
         - title: "Introduction"
           url: "index"
-        - title: "Route Params"
+        - title: "Paramètres des Routes"
           url: "route-params"
-        - title: "Service Injection"
+        - title: "Injection de Service"
           url: "service-injection"
         - title: "Ember Data"
           url: "ember-data"
-        - title: "Provider Components"
+        - title: "Composants Provider"
           url: "provider-components"
-        - title: "Recap"
+        - title: "Récap"
           url: "recap"
 
 - title: "Core Concepts"
   url: "toc-heading_core-concepts"
   is_heading: true
-- title: "Components"
+- title: "Composants"
   url: components
   pages:
-    - title: "Templates are HTML"
+    - title: "Les Templates sont du HTML"
       url: "index"
-    - title: "Introducing Components"
+    - title: "Introduction aux Composants"
       url: "introducing-components"
-    - title: "Component Arguments and HTML Attributes"
+    - title: "Arguments des Composants et Attributs HTML"
       url: "component-arguments-and-html-attributes"
-    - title: "Conditional Content"
+    - title: "Contenu Conditionnel"
       url: "conditional-content"
-    - title: "Block Content"
+    - title: "Contenu des Blocs"
       url: "block-content"
-    - title: "Helper Functions"
+    - title: "Fonctions Helper"
       url: "helper-functions"
-    - title: "Component State and Actions"
+    - title: "État des Composants et Actions"
       url: "component-state-and-actions"
-    - title: "Looping Through Lists"
+    - title: "Boucler sur des Listes"
       url: "looping-through-lists"
-    - title: Template Lifecycle, DOM, and Modifiers
+    - title: Cycle de Vie des Templates, DOM, et Modifiers
       url: template-lifecycle-dom-and-modifiers
-    - title: "Built-in Components"
+    - title: "Composants Built-in"
```

Comment:
```suggestion
    - title: "Composants Natifs"
```

```suggestion
    - title: "Composants Prédéfinis"
```

Reply:
To me, "Natif" tends to refer to native HTML, so I picked the other one.